optaplanner-core/src/main/java/org/optaplanner/core/api/score/Score.java

```diff
@@ -102,6 +102,22 @@
      */
     Score_ divide(double divisor);
 
+    /**
+     * Return a Score whose every level is a result of dividing the matching levels in this and the divisor.
+     * When rounding is needed, it should be floored (as defined by {@link Math#floor(double)}).
+     * <p>
+     * If the implementation has a scale/precision, then the unspecified scale/precision of the double divisor
+     * should have no impact on the returned scale/precision.
+     * <p>
+     * Allows division by zero if and only if both levels are zero, in which case the resulting level will also be zero.
+     * Otherwise {@link ArithmeticException} is thrown.
+     * @param divisor value by which this Score is to be divided
+     * @return this / divisor
+     */
+    default Score_ divide(Score_ divisor) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Public API change.
This is a default method, so that this is 100 % backwards compatible.